src/components/WeeklyForecast.svelte

```diff
@@ -0,0 +1,153 @@
+<script>
```

Comment:
![image](https://github.com/user-attachments/assets/cf7f5be3-a8de-4c74-af95-a6659864df08)
pleas check translation key for date